javascript/builtins/globals.json

```diff
@@ -709,49 +709,6 @@
             "deprecated": true
           }
         }
-      },
-      "uneval": {
```

Comment:
Can you add this to the PR title/description, or split into a separate PR?

Reply:
I'm adding it to the title. They have been removed in the same Firefox bug, so it makes sense to keep their removal together here.